cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -210,14 +217,20 @@ get_docker_default_runtime(){
 }
 
 get_docker_version(){
-	sudo docker version | awk '/Engine/{getline; print $2 }'
+	if [[ "$ID" =~ ^opensuse.*$ ]]; then
+		sudo docker version --format '{{.Client.Version}}' | cut -d '.' -f1-2
```

Comment:
I think you are only seeing client version because the docker service is stopped. I think we can use the client version for all distros to don't rely on docker service being up.